tests/dynamical/test_static_observation.py

```diff
@@ -30,29 +29,41 @@
 
 
 @pytest.mark.parametrize("model", [SimpleSIRDynamics()])
-@pytest.mark.parametrize(
-    "obs_handler", [PointObservation, NonInterruptingPointObservation]
-)
-def test_multiple_point_observations(model, obs_handler):
+def test_multiple_point_observations(model):
     """
     Tests if multiple PointObservation handlers can be composed.
     """
     S_obs = torch.tensor(10.0)
     data1 = {"S_obs": S_obs}
     data2 = {"I_obs": torch.tensor(5.0), "R_obs": torch.tensor(5.0)}
     with SimulatorEventLoop():
-        with obs_handler(time=3.1, data=data2):
-            with obs_handler(time=2.9, data=data1):
+        with PointObservation(time=3.1, data=data2):
+            with PointObservation(time=2.9, data=data1):
                 result = simulate(model, init_state, tspan)
 
     assert result.S.shape[0] == 5
     assert result.I.shape[0] == 5
     assert result.R.shape[0] == 5
 
 
+def _get_compatible_observations(obs_handler, time, data):
```

Comment:
This just lets us use the same code to test both `PointObservation` and the vectorized non-interrupting version.